pkg/fftm/manager.go

```diff
@@ -223,23 +230,26 @@ func (m *manager) fullScan() error {
 	var page int64
 	var read, added int
 	var lastOp *core.Operation
-	for {
-		ops, err := m.readOperationPage(lastOp)
-		if err != nil {
-			return err
-		}
-		if len(ops) == 0 {
-			log.L(m.ctx).Debugf("Finished reading all operations - %d read, %d added", read, added)
-			return nil
-		}
-		lastOp = ops[len(ops)-1]
-		read += len(ops)
-		for _, op := range ops {
-			added++
-			m.trackIfManaged(op)
+	for _, ns := range m.namespaces {
+		for {
+			ops, err := m.readOperationPage(ns, lastOp)
+			if err != nil {
+				return err
+			}
+			if len(ops) == 0 {
+				log.L(m.ctx).Debugf("Finished reading all operations for namespace '%s' - %d read, %d added", ns, read, added)
+				break
+			}
+			lastOp = ops[len(ops)-1]
+			read += len(ops)
+			for _, op := range ops {
+				added++
+				m.trackIfManaged(op)
+			}
+			page++
```

Comment:
Does `page` need resetting to `0` in the outer `ns` loop?

Reply:
Yes. Good catch, thanks.